openapi/values.yaml

```diff
@@ -16,6 +16,9 @@ properties:
       drbdVersion:
         type: string
         default: "9.2.5"
+      drbdVersionForceRebuild:
```

Comment:
I think we should remove this parameter and always perform the build if the version is lower than ours.

Reply:
We cannot remove this parameter, as "sort" continues to use it in checking the version. The check is done as "If the version is the same or lower, then do this or that". If we were to replace "sort" with bash functions, it would require hundreds of additional lines of code.